### PR TITLE
Add `electric_plant_depreciation_functional_ferc1` into the calc checking process

### PR DESCRIPTION
**Pull request by e-belfer**
Tiny PR to add the `electric_plant_depreciation_functional_ferc1` into the calculation checks pre-explosion for #2016. Addresses issue #2684.

# PR Checklist

- [x] Merge the most recent version of the branch you are merging into (probably `dev`).
- [ ] All CI checks are passing. [Run tests locally to debug failures](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#running-tests-with-tox)
- [x] Make sure you've included good docstrings.
- [x] Update the [release notes](https://catalystcoop-pudl.readthedocs.io/en/latest/release_notes.html) and reference reference the PR and related issues.
- [x] Do your own explanatory review of the PR to help the reviewer understand what's going on and identify issues preemptively.

### CODE REVIEW

File: src/pudl/transform/params/ferc1.py [hunk 1]
```diff
@@ -4065,6 +4065,11 @@
         "unstack_balances_to_report_year_instant_xbrl": {
             "unstack_balances_to_report_year": True
         },
+        "reconcile_table_calculations": {
+            "column_to_check": "ending_balance",
+            "calculation_tolerance": 0.2,
```
Comment (cmgosnell) [inline src/pudl/transform/params/ferc1.py:4070]: should the tolerance be this high? maybe a copy/paste from the other deprish table?

File: src/pudl/transform/ferc1.py [hunk 2]
```diff
@@ -1962,6 +1962,75 @@ def apply_xbrl_calculation_fixes(
                     },
                 ],
             },
+            "electric_plant_depreciation_functional_ferc1": {
+                # We use this name for the calculation but it gets renamed in
+                # `process_xbrl_metadata` to total.
```
Comment (cmgosnell) [inline src/pudl/transform/ferc1.py:1967]: hm yeah i think this is needed bc i never implemented a rename process for  `raw_xbrl_factoid_to_pudl_name` for the `unstack_balances_to_report_year_instant_xbrl` transform.

**Review by cmgosnell — APPROVED** [review]
can't believe this is so (relatively) straightforward

File: src/pudl/transform/params/ferc1.py (hunk 1 above)
Comment (e-belfer) [reply src/pudl/transform/params/ferc1.py:4070]: Not at all, this is a relic of my checking oops. Will delete and leave this as the default threshold.

**Pull request merged**